_posts/2018-09-05-understanding-the-promise-type.md

```diff
@@ -297,7 +297,7 @@ Once all of the parameters have been copied into the coroutine frame, the corout
 then constructs the promise object.
 
 The reason the parameters are copied prior to the promise object being constructed
-is to allow the promise object to be given access to the post-copied parameters
```

Comment:
The change here is incorrect.
While the promise_type's operator new() is provided a reference to the pre-copied parameters, the promise_type's constructor is provided references to the post-copied parameters.